cmd/ocm-backplane/remediation/remediation.go

```diff
@@ -103,12 +103,15 @@ func runCreateRemediation(args []string, clusterKey string, urlFlag string) erro
 		return err
 	}
 
-	backplaneHost := bpConfig.URL
+	bpURL := globalOpts.BackplaneURL
```

Comment:
Also there were some issue with setting the correct config options. Can we maybe start tracking something ( maybe already tracked? ) to improve, how its used. I think any common config, flags, env, should be dealt with before going to a specific command. 

So that we do not have to deal with bpURL, backplaneapi client, etc. in each command?

Reply:
just for info @RaphaelBut , We should completely get rid of config/cmd options  backplanURL and we should always get it via OCM API 
https://github.com/openshift/backplane-cli/blob/main/pkg/cli/config/config.go#L254